pkg/registry/config/types.go

```diff
@@ -0,0 +1,15 @@
+package config
+
+// Config is the config for ormb.
+type Config struct {
+	Username string `json:"username,omitempty"`
+	Password string `json:"password,omitempty"`
+	Domain   string `json:"domain,omitempty"`
+
+	KubeConfig string `json:"kube_config,omitempty"`
+}
+
+// New create a new Config.
+func New() *Config {
```

Comment:
The config's value need init

Reply:
Do you mean setting default values for the options?

Reply:
In `main.go`,  you call `registryconfig.New()` function, but the config object is no value, how to get the value?

Reply:
It will be set by nirvana.

Reply:
oh, got it, LGTM